DESCRIPTION

```diff
@@ -12,7 +12,8 @@ BugReports: https://github.com/Rdatatable/data.table/issues
 VignetteBuilder: knitr
 ByteCompile: TRUE
 Authors@R: c(
-  person("Matt","Dowle",           role=c("aut","cre"), email="mattjdowle@gmail.com"),
+  person("Tyson","Barrett",        role=c("aut","cre"), email="t.barrett88@gmail.com"),
+  person("Matt","Dowle",           role="aut",          email="mattjdowle@gmail.com"),
```

Comment:
I'm ok for going ahead on master branch with Tyson as CRAN maintainer, for next major feature release (non-patch).
Also for patch releases, I was assuming that Matt will continue doing that, until governance is decided.
did anybody email Matt about this? (I did not) Matt has to give his OK to CRAN maintainers on email, in order for this to be updated on CRAN.

Reply:
1.14.10 branch already has that change: https://github.com/Rdatatable/data.table/compare/1.14.8...hotfix1.14.10
so we may need to revert that change possibly.

From what you are suggesting it most likely means that Matt would have to still make 2 pkg submissions to CRAN: 1.14.10 and then 1.15.0. Because the latter one would be moving maintainership to Tyson.